plugins/com.google.cloud.tools.eclipse.appengine.libraries/schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd

```diff
@@ -0,0 +1,194 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.appengine.libraries" xmlns="http://www.w3.org/2001/XMLSchema">
```

Comment:
How does Eclipse use this schema? What happens if it is not present? If nothing changes, then we can simply remove it.